ClientRuntimes/CSharp/ClientRuntime.Azure.Authentication/UserTokenProvider.cs

```diff
@@ -261,33 +261,27 @@ public class UserTokenProvider : ITokenProvider
         public static async Task<ServiceClientCredentials> LoginWithPromptAsync(string domain, ActiveDirectoryClientSettings clientSettings, 
```

Comment:
@haocs I wonder if we're going to need to change this signature cross-platform.  I suspect we will need to expose PlatformProvider, I'm not sure if the default PlatfromProvider will work cross-platform.  We should ask the ADAL team about this.